ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-device-documentation.j2

```diff
@@ -105,6 +105,8 @@
 {% include 'documentation/ip-dhcp-relay.j2' %}
 {# IPv6 DHCP Relay #}
 {% include 'documentation/ipv6-dhcp-relay.j2' %}
+{# IPv6 Neighbors #}
```

Comment:
wouldn't it make sense to put this IPv6 Neighbor section into the routing section, like ARP for IPv4?